docs/manual/nodes.md

```diff
@@ -535,6 +535,63 @@ topology:
         net.ipv6.icmp.ratelimit: 1000
 ```
 
+### waitFor
+For the explicit definition of startup dependencies between nodes, the `waitFor` knob under the `kind` or `node` level can be used.
+
+Node _srl3_ will wait until _srl1_ and _srl2_ are created before it is created. The _linuxNode_ will via the definition in the _linux_ kind wait for all three _srl_ nodes to be created before it gets created.
```

Comment:
can you change `linuxNode` to `client`. Makes it easier to read I think

Reply:
done!